src/core/system.conf.in

```diff
@@ -27,6 +27,7 @@
 #CrashChangeVT=no
 #CrashShell=no
 #CrashReboot=no
+#CrashExit=no
```

Comment:
I think we should just add CrashAction= instead, that takes reboot/exit/shell

Reply:
Hmm, maybe this shouldn't cover `crash_shell`, but I'd love to add `CrashAction=poweroff` while at it too. This also makes the semantics closer to `SuccessAction=` and friends.

Plus, these are mutually exclusive, and one option really makes more sense IMO.

Reply:
`CrashExit=yes` is identical to to `CrashAction=poweroff`. I'm happy to rename it to `CrashPoweroff=` instead if that's preferred.

Reply:
Yeah, `exit` is a bit misnomer.

But still, I think we want to have something like `SuccessAction=` here. One day people might come and add `kexec`, and I think it's better to make this generic here rather than later...